mathesar/api/ui/viewsets/version.py

```diff
@@ -0,0 +1,50 @@
+import requests
+
+from rest_framework import viewsets
+from rest_framework.decorators import action
+from rest_framework.response import Response
+
+from mathesar import __version__
+# TODO this import path is pretty long; might want to remove redundant occurances of "_exceptions"
+from mathesar.api.exceptions.version_exceptions.base_exceptions import GithubReleasesAPIException
+
+
+class VersionViewSet(viewsets.ViewSet):
+
+    @action(methods=['get'], detail=False)
+    def current(self, _):
+        current_version = __version__
+        return Response(current_version)
+
+    @action(methods=['get'], detail=False)
+    def latest(self, _):
+        latest_version = _get_latest_release_tag_name_from_github()
+        return Response(latest_version)
+
+
+def _get_latest_release_tag_name_from_github():
+    """
+    If Github API returns a non-200 response, will raise and exception with the status code and
+    body of Github API's response.
+    """
+    owner = 'centerofci'
+    repo = 'mathesar'
+    url = f'https://api.github.com/repos/{owner}/{repo}/releases/latest'
+    headers = {
+        'Accept': 'japplication/vnd.github+json',
```

Comment:
I made a commit to correct the Accept header to be `application/vnd.github+json` instead of `'japplication/vnd.github+json'` as all the Github API docs point to that. Please revert it back if it is not correct.